src/api/spec/support/database_cleaner.rb

```diff
@@ -12,7 +12,7 @@
     # Before the test suites runs make sure the database is empty
     log_level = Rails.logger.level
     Rails.logger.level = :fatal
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:transaction)
```

Comment:
this doesn't make sense, sorry. The code tries to cleanup the initial database - and you simply can't clean up existing data with a transaction.

The code here is a little conservative and for your use case it's most likely preferable to just remove it completly by means of an environment variable.